src/test/scala/io/iohk/ethereum/jsonrpc/EthServiceSpec.scala

```diff
@@ -105,6 +105,70 @@ class EthServiceSpec extends FlatSpec with Matchers with ScalaFutures with MockF
     response.transactionResponse shouldBe Some(expectedTxResponse)
   }
 
+  //FIXME: This test should be changed once the pending block doesn't equal to the latest one
```

Comment:
We should probably address it soon (there's currently an open task for that https://iohk.myjetbrains.com/youtrack/issue/EC-184) but I think it will be better to handle this in a separate PR.